src/exo/platforms/sve_vls.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from exo import DRAM, Memory, instr
+from exo.core.memory import MemGenError
+
+
+def _is_const_size(sz, c):
+    return sz.isdecimal() and int(sz) == c
+
+
+def _sve_vector_init(sve_vector_bits):
+    assert sve_vector_bits >= 128
+    assert sve_vector_bits <= 2048
+    assert sve_vector_bits % 128 == 0
+
+    vec_types = {
+        "float": (sve_vector_bits // 32, "svfloat32_vls_t"),
+        "double": (sve_vector_bits // 64, "svfloat64_vls_t"),
+    }
+
+    class SVE_VLS(Memory):
+        @classmethod
+        def global_(cls):
+            return f"""#include <arm_sve.h>
+typedef svfloat32_t svfloat32_vls_t __attribute__((arm_sve_vector_bits({sve_vector_bits})));
```

Comment:
The reason you're getting a Codecov warning is that you don't have tests that compile the scheduled Exo code.
Can you use `compiler` (refer to test_codegen.py) to generate C code and add that to golden as well? We want to test the generated C code as well as Exo IR.

Reply:
@eyr1n lmk if you have time to make this change! The PR looks good otherwise.